lib/v2/screens/passcode/interactor/viewmodels/passcode_state.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:equatable/equatable.dart';
+import 'package:meta/meta.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+
+/// STATE
+class PasscodeState extends Equatable {
+  final PageState pageState;
+  final bool isCreateView;
```

Comment:
Can we make this an Enum? 

CreateView
CreateMode

Reply:
actually, i am not sure how createMode is used.

Reply:
It cant be a enum they (isCreateView, isCreateMode) have a differente purpose.

- isCreateView is to condition the current view of passcode.

- isCreateMode is to know that it is still in creation mode even if the screen is VerifyPasscode, so knowing the bloc is in creation mode can take other instructions in its coditions.